internal/tiltfile/docker.go

```diff
@@ -590,6 +591,11 @@ func (s *tiltfileState) dockerignoresForPaths(paths []string) []model.Dockerigno
 			continue
 		}
 
+		result = append(result, model.Dockerignore{
+			LocalPath: path,
+			Contents:  ignoresToDockerignoreContents(ignores),
```

Comment:
afaict, this func can return however many dockerIgnores you want, and each elem of `ignore` is a complete `Dockerignore` in itself -- why bother to write all the `ignores` to one file?

i.e. why not:
```
for _, ignore := range ignores {
    result = append(result, model.Dockerignore{
        LocalPath: path,
        Contents: ignore,
    }
}
```

Reply:
No real reason, but you made me realize that we can create the contents once outside of the for loop because it's the same for every path, which might be getting at some of your unease. Let me try that.